ol3-cesium-plugin/garastersynchronizer.js

```diff
@@ -0,0 +1,81 @@
+goog.provide('ga.GaRasterSynchronizer');
+goog.require('olcs.RasterSynchronizer');
+
+
+
+/**
+ * Handle mapping OL3 2D layers to Cesium 3D Imageries in EPSG:4326.
+ * @param {!ol.Map} map
+ * @param {!Cesium.Scene} scene
+ * @constructor
+ * @extends {olcs.RasterSynchronizer}
+ * @api
+ */
+ga.GaRasterSynchronizer = function(map, scene) {
+  goog.base(this, map, scene);
+};
+goog.inherits(ga.GaRasterSynchronizer, olcs.RasterSynchronizer);
+
+
+/**
+ * @override
+ */
+ga.GaRasterSynchronizer.prototype.convertLayerToCesiumImagery =
+    function(olLayer, viewProj) {
+
+  /**
+   * @type {Cesium.ImageryProvider}
+   */
+  var provider = null;
+  var source = olLayer.getSource();
+
+  if (source instanceof ol.source.Vector) {
+    return null;
+  }
+
+  // Read custom, non standard properties
+  var layerId = olLayer['id'];
+  var layerTime = olLayer['time'];
+
+  if (source instanceof ol.source.WMTS) {
+    provider = new Cesium.WebMapTileServiceImageryProvider({
+      url: 'https://mf-chsdi3.dev.bgdi.ch/mom_fix_1594/1.0.0/' +
+         layerId + '/{Style}/' + layerTime +
+         '/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png',
```

Comment:
Usually, we have environment specific configuration in [application controllers](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/js/FeaturetreeController.js#L19) or simply on the `gaGlobalOptions` to avoid having the in the components.

Is it easily possible to do the same for ol3-cesium plugins?